docker/optimism/optimism.sh

```diff
@@ -50,7 +50,8 @@ create_deployment_config_file() {
 
 e2e_test() {
     l2_rpc=http://localhost:8545
-    poetry run python "${ROOT}/scripts/e2e-test.py" ${DEPLOYMENT_DIR} ${KEYFILE} ${l2_rpc}
+    password=""
```

Comment:
```suggestion
    password="''"
```

Reply:
fyi that doesn't work. It puts the string `''` as the password. 

Quoting the parameter did the job `..... "${password}" ...`